features/step_definitions/bootstrap_steps.rb

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2010-2017 SUSE-LINUX
```

Comment:
we can put this step in common.steps or command.steps.

We basically don't need to create a new step file for the bootstrap.